components/brave_rewards/resources/extension/brave_rewards/manifest.json

```diff
@@ -55,6 +56,14 @@
       ]
     }
   ],
+  "externally_connectable": {
+    "ids": [
+      "*"
```

Comment:
is this intentional? it allows any extension or app to connect to the brave_rewards extension.

Reply:
is this needed because greaselion doesn't have a fixed ID? i could see that causing a lot of security issues.

Reply:
Yes, it's due to the dynamic nature of the Greaselion extension IDs.

Reply:
Looks like we have two threads going for this one, but it's removed now :-)